hugo/content/documentation/_index.md

```diff
@@ -1,9 +1,8 @@
----
-title: Mumble Documentation
-date: 2019-10-22
----
-The Mumble project consists of many things, and so the documentation has separate content for each concern and sub-project.
-
-Most of our documentation is in our Wiki at [wiki.mumble.info](https://wiki.mumble.info/wiki/Main_Page). We are in the process of moving (some) of that documentation onto this website, here under [www.mumble.info/documentation]({{< relref "/documentation" >}}).
-
-If you do not find what you are looking for here then please refer to the [documentation wiki](https://wiki.mumble.info/wiki/Main_Page) and navigate or search for what you are looking for there.
+---
+title: Mumble Documentation
+date: 2020-12-22
+---
+
+For now you find most of our documentation in our Wiki at [wiki.mumble.info](https://wiki.mumble.info/wiki/Main_Page). 
+
+We are currently working on newer and more comprehensive documentation, that will be placed directly on the website.
```

Comment:
Why is every line in the file changed? Line endings?

Reply:
Indeed the title section should have remained unchanged :thinking:

Reply:
Well, it's only the one line that should in theory remain unchanged, why is that now a problem?

Reply:
Because this means that you probably messed up the line-endings.
I just checked that the original version seems to have Unix line endings (as it should). Thus I assume that you introduced windows line endings when editing the file :point_right: 
Could you check that please?

Reply:
file says:
`_index.md: ASCII text`, so it should be Unix (afaik).

Reply:
Uhm usually ASCII has nothing to do with line endings...
Are you on Windows or Linux? On Linux there is this neat tool called `dos2unix` that'll convert line endings to unix style

Reply:
I am on linux.
The internet told me that linux text would show `ASCII text` while windows would show: `ASCII text, with CRLF line terminators`.

Reply:
Okay so I just checked again and you did change the line endings but in the other way than I suspected. They appear to be Windows style in the current version and you changed it to Unix style.

Thus I'm okay with that. Sorry for the confusion :see_no_evil: 

@Kissaki any further objections to merging this PR?

Reply:
Can we merge that now, it's only a small change, basically removing the double information sentence that is there now.
Last post was 26 days ago.

Reply:
Yep